### DISCUSSION TITLE
Getting Node version error with no reason

### DISCUSSION THREAD

**Discussion by bardaxx** [vercel/vercel#5988] [Help]
I'm getting:

The engine "node" is incompatible with this module. Expected version "14.15.5". Got "14.15.4"
-- | --

 | error Found incompatible module.
 
 Error: Command "yarn install" exited with 1

but the node version is the right one. Any help?

**Comment by PiotrRut** [comment]
@giacomoalonzi Did you have any luck with this? Having the same issue now, my project needs node `^14.16` but Vercel refuses to use it
